dashboard/config/scripts/levels/Allthethings Java Lab 1.level

```diff
@@ -1,12 +1,16 @@
 <Javalab>
   <config><![CDATA[{
-  "game_id": 68,
-  "created_at": "2021-02-15T20:07:34.000Z",
-  "level_num": "custom",
-  "user_id": 1,
   "properties": {
+    "encrypted": "false",
+    "instructions_important": "false",
+    "short_instructions": "This is a sample Java Lab level.",
```

Comment:
Please don't use short instructions for CSA. Short instructions are only for CSF style levels. You will want to use long instructions for CSA and hide the short instructions area for CSA.

Reply:
Here is the line to hide short instructions : https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/editors/fields/_instructions.haml#L5

Reply:
You will also want to hide the short instructions area for TTS here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/editors/fields/_tts.haml#L7-L10